storage/Devices/BcacheImpl.cc

```diff
@@ -407,6 +407,21 @@ namespace storage
     }
 
 
+    void
+    Bcache::Impl::remove_bcache_cset()
+    {
+	if(get_type() == BcacheType::FLASH_ONLY)
+	    ST_THROW(LogicException("A Caching Set cannot be detached from a Flash-only Bcache"));
```

Comment:
Here the function is called "remove_..." but the error message says "detach". Please be consistent with the terms (I suggest to follow the bcache project).

Reply:
I will adapt the messages.